dtfs-central-api/src/v1/routes/utilisation-reports-routes.js

```diff
@@ -169,4 +170,37 @@ utilisationReportsRouter
   .route('/reconciliation-details/:reportId')
   .get(validation.sqlIdValidation('reportId'), handleExpressValidatorResult, getUtilisationReportReconciliationDetailsById);
 
+/**
+ * @openapi
+ * /utilisation-reports/:id/selected-fee-records-details:
+ *   get:
+ *     summary: Get the fee record details for the selected fee record ids
+ *     tags: [UtilisationReport]
+ *     description: Get the fee record details for the selected fee record ids
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: the id for the report the fees belong to
+ *     responses:
+ *       200:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               $ref: '#/definitions/SelectedFeeRecordsDetails'
+ *       400:
+ *         description: Bad request
+ *       404:
+ *         description: Not Found (if the report with matching id cannot be found)
+ *       500:
+ *         description: Internal Server Error
+ */
+utilisationReportsRouter
+  .route('/:id/selected-fee-records-details')
+  .get(validation.sqlIdValidation('id'), handleExpressValidatorResult, getSelectedFeeRecordDetails);
```

Comment:
I think it might be fair in this case for the selected fee record ids to be stored in the query rather than the `GET` request body. My reasoning is that this will only ever be an array of numbers, so it's probably a lot safer for us to assume that we won't reach the URL length limit even if we select many fee records than it would be if we were using an object, for instance. I also think the documentation above needs to be updated to either include the request body if we decide to keep that or the query if we move to that :)

Reply:
Hmm I hadn't really considered quite how much shorter this would be than the original page request, and agree it will be a fair bit shorter. I do however still think it is safer to do it this way, and has also now already been implemented this way 😅 so I am not too keen to change it unless you think there is a good reason to do so, especially as it is less safe

Reply:
From looking online there seems to be many hot takes from both the `GET request shouldn't use body` and `GET request should use body` sides, so I'm happy to go with it. An interesting concern I read about was that you may end up with less security as HTML servers typically aren't set up to handle `GET` requests with a body, so things may sneak through checks that would normally get picked up in query or path parameters. I don't know how relevant this is for modern servers though, so I'm happy to keep the `GET` request body here